src/meltano/core/venv_service.py

```diff
@@ -738,28 +738,13 @@ async def _write_error_log(self, err: AsyncSubprocessError) -> None:
         """Write error details to the log file."""
         import anyio
 
-        stderr_content = await err.stderr
-        timestamp = datetime.now(timezone.utc).isoformat()
-        python_path = self.exec_path("python")
-
         async with await anyio.open_file(
-            self.pip_log_path, "a", encoding="utf-8"
+            self.pip_log_path,
+            "a",
+            encoding="utf-8",
         ) as log_file:
-            await log_file.write(
-                f"\n--- Installation attempt failed at {timestamp} ---\n"
```

Comment:
I'll ask the Astral folks if they'd be willing to add the timestamp to the logs. Seems possible according to

- https://docs.rs/tracing-subscriber/latest/tracing_subscriber/index.html#feature-flags